src/react/components/GenericForm.jsx

```diff
@@ -10,6 +11,49 @@ import 'antd/es/input/style/index.js';
 
 const { TextArea } = Input;
 
+const zIndexValue = 999999;
```

Comment:
```suggestion
// If the form is shown on popup the Select dropdown opens behind popup without this
// FIXME: this will probably not work with modal popups (dropdown might be over the modal overlay)
const zIndexValue = 999999;
```